core/src/datayoga_core/expression.py

```diff
@@ -111,9 +115,13 @@ def compile(self, expression: str):
         # they will get parsed and binded for performance instead of traversing the entire payload
         self._column_names = set()
         for _exp in self._fields.values():
-            self._column_names.update(
-                [tuple(column.sql().replace('"', "").split("."))
-                 for column in sqlglot.parse_one("SELECT " + _exp.replace('`', '"')).find_all(sqlglot.exp.Column)])
+            try:
+                self._column_names.update(
+                    [tuple(column.sql().replace('"', "").split("."))
+                     for column in sqlglot.parse_one("SELECT " + _exp.replace('`', '"')).find_all(sqlglot.exp.Column)])
+            except Exception as pe:
```

Comment:
shouldn't we catch the ParseError you imported?

Reply:
no, since it sometimes throws runtimeerror and others